documentation/builders/README.md

```diff
@@ -19,6 +18,8 @@
     * [OnOff SHIM for safe power on/off](./components/power/onoff-shim.md)
 * [Soundcards](./components/soundcards/)
     * [HiFiBerry Boards](./components/soundcards/hifiberry.md)
+* [Synchronisation](./components/synchronisation/)
```

Comment:
Is it ok to add all components here, or do we need to split this for hardware and software implementations?

Reply:
As soon as this list grows, we can slit it. I like the idea

Reply:
How about splitting it now and restructure a bit.
In my opinion its not really clear what the different headers mean (config, components, advanced).

Idea for restructuring:
Additionally i would suggest to add the rfid reader docs for supported readers. Maybe even move to builder space (except mock and template)?

![Bildschirmfoto 2024-01-24 um 08 37 39](https://github.com/MiczFlor/RPi-Jukebox-RFID/assets/103769832/0170b2a8-4f0f-447f-b5b2-2c4800c8da56)